pandas/tests/test_groupby.py

```diff
@@ -2481,6 +2481,26 @@ def test_size(self):
             self.assertEqual(result[key], len(group))
 
     def test_count(self):
+        from string import ascii_lowercase
+        n = 1 << 15
+        dr = date_range('2015-08-30', periods=n // 10, freq='T')
+
+        df = DataFrame({
+                '1st':np.random.choice(list(ascii_lowercase), n),
+                '2nd':np.random.randint(0, 5, n),
+                '3rd':np.random.randn(n).round(3),
+                '4th':np.random.randint(-10, 10, n),
+                '5th':np.random.choice(dr, n),
+                '6th':np.random.randn(n).round(3),
+                '7th':np.random.randn(n).round(3)})
+
+        for col in '3rd', '5th', '6th', '7th':
+            df.loc[np.random.choice(n, n // 10), col] = np.nan
+
+        for key in '1st', '2nd', ['1st', '2nd']:
+            left = df.groupby(key).count()
+            right = df.groupby(key).apply(DataFrame.count).drop(key, axis=1)
+            assert_frame_equal(left, right)
```

Comment:
no i do not; this is a random test. both [`test_groupby.py:L2483`](https://github.com/pydata/pandas/blob/81b647f4e623446d7b584e216382e79c2ed2e720/pandas/tests/test_groupby.py#L2483) and [`test_frame.py:L12613`](https://github.com/pydata/pandas/blob/81b647f4e623446d7b584e216382e79c2ed2e720/pandas/tests/test_frame.py#L12613) already include hand written tests.